apps/front/jest.config.js

```diff
@@ -7,9 +7,12 @@ module.exports = {
     resetModules: true,
     verbose: true,
     transform: {
+        '^.+\\.svelte$': 'jest-transform-svelte',
```

Comment:
C'est quoi la différence entre `jest-transform-svelte` et `svelte-jester` ?

Reply:
Bonne question ^^ Plus d'utilisateurs sur npm pour `jest-transform-svelte` ... Après, la doc semble dire que `svelte-jester` doit être utiliser avec `Testing Library` !? Mais cela semble tout de même fonctionner avec `jest-transform-svelte`. Mmmm vraiment jeune Svelte encore. Restons comme ça, en ce notant dans un coin de la tête que si on a des problèmes mystique durant les tests, on pourrait switcher de lib.

Reply:
C'est noté !